yarn.lock

```diff
@@ -276,6 +276,13 @@ ansi-wrap@0.1.0:
   version "0.1.0"
   resolved "https://registry.yarnpkg.com/ansi-wrap/-/ansi-wrap-0.1.0.tgz#a82250ddb0015e9a27ca82e82ea603bbfa45efaf"
 
+anymatch@^1.3.0:
```

Comment:
Not super keen on all of these additions. :unamused:

But what're y'gonna do? :man_shrugging: